Makefile

```diff
@@ -111,3 +111,23 @@ submodules:
 	@git submodule update --init --recursive
 
 .PHONY: manifests cobertura reviewable submodules fallthrough
+
+# ====================================================================================
+# Special Targets
+
+define CROSSPLANE_HELP
+Crossplane Targets:
+    manifests          Generate manifests e.g. CRD, RBAC etc.
+    cobertura          Generate a coverage report for cobertura applying exclusions on generated files.
+    reviewable         Ensure a PR is ready for review.
+    test-integration   Run integration tests.
+    submodules         Update the submodules, such as the common build scripts.
+endef
+export CROSSPLANE_HELP
+
+crossplane.help:
+	@echo "$$CROSSPLANE_HELP"
+
+help-special: crossplane.help
+
+.PHONY: crossplane.help
```

Comment:
Would it make sense to declare `help-special` as `.PHONY` too? I know it's declared in `common.mk`, but for readability I'm thinking it may make sense to include it here too.